spec/02-integration/04-admin_api/03-consumers_routes_spec.lua

```diff
@@ -247,6 +247,11 @@ describe("Admin API (#" .. strategy .. "): ", function()
           pages[i] = json
         end
       end)
+      it("not crashed by empty custom_id #KAG-867", function()
+        local res = client:get("/consumers?custom_id=")
+        local body = assert.res_status(200, res)
```

Comment:
Why is this not a 400?

Reply:
You're right. It's an incorrect arg instead of omitted arg.